rating/peersRatingHandler.go

```diff
@@ -75,6 +93,8 @@ func (prh *peersRatingHandler) AddPeer(pid core.PeerID) {
 	}
 
 	prh.topRatedCache.Put(pid.Bytes(), defaultRating, int32Size)
+	prh.updateRatingsMap(pid, defaultRating, 0)
+	prh.updateMetrics()
```

Comment:
Somehow related question: when and how do we remove a peer from this rating component?

Reply:
as discussed, pushed the changes needed to clean cachers